.github/workflows/update-maintainers.yml

```diff
@@ -0,0 +1,254 @@
+name: Update maintainers list
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - lib/maintainers.nix
+  schedule:
+    # Update every Monday at 9 AM UTC
+    - cron: "0 9 * * 1"
+  workflow_dispatch:
+    inputs:
+      create_pr:
+        description: "Create PR even if no changes"
+        required: false
+        default: false
+        type: boolean
+
+jobs:
+  update-maintainers:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nix-community' || github.event_name == 'workflow_dispatch'
+    # Permissions required for workflow
```

Comment:
Nit: this makes it sound like this is the canonical place where the permissions are defined. However we actually have them in two places, and this is the less important of the two since it is only used on forks that don't have an APP_ID.

Reply:
>These dictate the permissions of github.token and are entirely unrelated to the app token.
>> the others are just to limit the scope of permissions granted to an app.

Understood, as part of previous response.

Guess its differing opinion on what's `important`. I think declaring the app token permissions as less because they are optional for functionality. But, the app token permissions are just a couple lines lower with the exact same naming convention and didn't think it warranted copy/paste for same explanation.